Dockerfile

```diff
@@ -1,19 +1,11 @@
-# WARNING: This Dockerfile, and the associated docker-compose.yml,
-# are only functional for Cypress v3.x. Docker installation 
-# methods are not yet supported for Cypress v4, as of 2018-07-10
-
-FROM phusion/passenger-ruby23:latest
+# Pinned to the latest ruby 2.6.0 version of the Passenger base Docker image
+FROM phusion/passenger-ruby26:1.0.1
```

Comment:
Do you mind expanding the comment here to say when we can actually bump to a newer version, is there a specific issue blocking us from using latest? Is this just due to the fact that we're using a slightly out-of-date version with Cypress?

Reply:
The only restriction here is that we need to use a version of `passenger-ruby26` that matches the `.ruby-version` version of Ruby (at the moment, 2.6.0). Phusion only tracks minor version bumps in package names, so we can't bump to `1.0.2` (or `latest`) until we upgrade Cypress to 2.6.1.

Reply:
Sounds good, once this is merged we should make a new PR to bump ruby to 2.6.1 across the board.